tilewe/engine.py

```diff
@@ -33,22 +34,27 @@ def on_search(self, board: tilewe.Board, _seconds: float) -> tilewe.Move:
 
 class Tournament: 
 
-    def __init__(self, engines: list[Engine]): 
-        self.engines = list(engines) 
+    def __init__(self, engines: list[Engine], move_seconds: int=60): 
+        self.engines = list(engines)
+        self.move_seconds = move_seconds
```

Comment:
Didn't want to have to do this necessarily, but since `_play_game` can only take one argument due to `imap` limitations the information needs to pass through the class. It seems reasonable to allow `play` to do this, because you may want to call subsequent `tournament.play` rounds without re-initializing.